modules/gsp-namespace/main.tf

```diff
@@ -0,0 +1,32 @@
+resource "tls_private_key" "github_deployment_key" {
+  algorithm = "RSA"
+  rsa_bits  = "4096"
+}
+
+/* resource "aws_iam_role" "dev" { */
```

Comment:
TODO: Remove commented code.

Reply:
Coming back to this in the next stage of refactoring